xcodeproj/internal/default_input_file_attributes_aspect.bzl

```diff
@@ -87,6 +91,9 @@ def _default_input_file_attributes_aspect_impl(target, ctx):
             "runtime_deps": [target_type.compile],
             "data": [target_type.resources],
         }
+    elif ctx.rule.kind == "objc_import":
+        xcode_targets = {}
+        excluded = ("archives")
```

Comment:
All changes in here are temporary, until this is rewritten to default to under-collecting instead of over-collecting.